### PR TITLE
VDA 5050 Connector for mobile transport robots in intralogistics

### PR DESCRIPTION
**Pull request by frothm**
# Please Add This Package to be indexed in the rosdistro.

melodic vda5050_connector
noetic vda5050_connector

# The source is here:

https://github.com/tum-fml/ros_vda5050_connector.git

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

**Comment by methylDragon** [comment]
Note that since this package was not bloomed, it won't be available to `apt install` and won't build on the buildfarm. Is the intention just to get the docs and source indexed?

**Review by clalancette — APPROVED** [review]

File: melodic/distribution.yaml [hunk 1]
```diff
@@ -14811,6 +14811,16 @@ repositories:
       url: https://github.com/anybotics/variant-release.git
       version: 0.1.6-1
     status: maintained
+  ros_vda5050_connector:
```
Comment (clalancette) [inline melodic/distribution.yaml:14814]: According to CI, this is now alphabetically in the wrong place.  Also, this should match the name of your package, so this really should be `vda5050_connector`.  Same goes for the noetic side.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: melodic/distribution.yaml (hunk 1 above)
Comment (clalancette) [reply melodic/distribution.yaml]: So the thing is that the new name is not [REP-144 compliant](https://www.ros.org/reps/rep-0144.html), particularly the 3rd global rule which says:

```
A package name should not contain ros as it is redundant. Exceptions include core packages and ROS bindings of an upstream library (e.g. moveit_ros)
```

I guess the question is whether this qualifies for an exception?

Comment (frothm) [reply melodic/distribution.yaml]: I totally agree... It is difficult making all those marketing/PR people behind me happy.

I now convinced my team to stick with the previous, conform name for the ROS package, but rename the git repository. I hope this is acceptable, even though the package name now differs from the repo name...

**Review by clalancette — APPROVED** [review]

**Pull request merged**